index.html

```diff
@@ -2810,6 +2813,84 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section id="solid-color-icons">
+        <h2>
+          Solid color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a single color
+          or gradient, where only the transparency of the icon can be
+          controlled. As web applications should work across multiple
+          platforms, it it possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>monochrome</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>monochrome</a> icon, the user agent MUST NOT use
+          the red component, green component, or blue component of a pixel. If
+          it has alpha equal to zero, the user agent SHOULD NOT display it.
+          If it the alpha component is greater than zero, the user agent SHOULD
+          display it with any tint.
+        </p>
+        <p>
+          If no monochrome icon is defined when an icon with a solid icon must
+          be presented, the user agent MAY show a fallback icon.
+        </p>
+        <p class="note">
+          Designers of <a>monochrome</a> icons are recommended to set all
+          pixels to black and use transparency to create a silhouette of their
+          icon.
+        </p>
+        <p>
```

Comment:
Do we want to allow for inversion as well? Windows currently does this when switching between light & dark mode.

Reply:
The intention is that the spec allows for the icon to be displayed in any color (including white). This is just a recommendation that developers save the asset as an all-black image. The user agent can tint it afterwards.

Reply:
Github assigned the comment to a line, which made this slightly confusing. I was not saying this was for authors, but rather for implementors to give them the flexibility to adapt the icon as necessary.

Reply:
By inversion, are you referring to making the solid areas transparent and the transparent areas solid? If so I agree that would be good to add in.

If you mean replace black with white, then I try to express this in the spec by indicating that the r,g,b values can be replaced by the user agent. Do you have any suggestions for making this more clear?

Reply:
I think both of these are already implied by the spec. If a user agent wanted to, say, display the icon as pink-on-green, it would make the pixels pink, and then display the icon on a green background. If it wanted to invert that color scheme, it would make the pixels green and then display the icon on a pink background.

The only thing that wouldn't be supported would be if the UA, for some reason, wanted to make the "transparent" part of the image a solid colour (say, pink), and then have the "opaque" part of the image show through some background underneath. I would say we explicitly do not want to support that, since then we're kinda messing with the brand logos in a way they don't necessarily want.